bears/python/PEP8Bear.py

```diff
@@ -41,14 +42,69 @@ def run(self, filename, file,
                    'max_line_length': max_line_length,
                    'indent_size': indent_size}
 
+        errors = self.list_pep8_errors(source=file, options=options)
```

Comment:
`autopep8.fix_code` method fixes the errors reported by `pycodestyle`.
If you check `autopep8`'s code you will see that it runs a `pycodestyle` check and then it applies the fixes on the reported errors.